doc/appendix-de-kassensichv/data-structures/data-structures.md

```diff
@@ -1,19 +1,19 @@
 ## Data Structures
 
-This chapter expands on the descriptions of the data structures covered in the Chapter ["Data Structures"](../../general/data-structures/data-structures.md) of the General Part, with country specific information applicable to the German market.
+This chapter expands on the descriptions of the country-specific covered in the Chapter ["Data Structures"](../../general/data-structures/data-structures.md) of the General Part, with country specific information applicable to the German market.
 
 ### Receipt Request
 
 #### Single fields
 
-Following chapter highlights fields from the receipt request that need a special handling for the german market.
+Following chapter highlights fields from the receipt request that need special handling for the German market.
```

Comment:
unless I misunderstand the context here, IMO it should be
The chapter highlights fields from the receipt request that need special handling for the German market are listed below:

Reply:
hm, i think i know what you mean